clustered/client/build.gradle

```diff
@@ -45,4 +44,5 @@ dependencies {
   }
   testImplementation 'org.xmlunit:xmlunit-core:2.6.0'
   testImplementation 'org.xmlunit:xmlunit-matchers:2.6.0'
+  testImplementation ("org.terracotta:statistics:$parent.statisticVersion")
```

Comment:
Could we refactor the `StatisticsTestUtils` class to depend on the `StatisticsService` and then eliminate this dependency?

Reply:
I assume that's a no then?

Reply:
It directly uses `ContextManager` to get `OperationStatistic` as below
`for (final TreeNode statNode : ContextManager.nodeFor(store).getChildren())`